presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -336,24 +337,24 @@ Property Name                                Description
                                              API, the Hadoop configuration will be passed in after
                                              the object has been created.
 
-``hive.s3.upload-acl-type``                  Canned ACL to use while uploading files to S3 (defaults
-                                             to ``Private``).
+``hive.s3.upload-acl-type``                  Canned ACL to use while uploading files to S3, defaults
+                                             to ``Private``.
 
 ``hive.s3.skip-glacier-objects``             Ignore Glacier objects rather than failing the query. This
-                                             will skip data that may be expected to be part of the table
+                                             skips data that may be expected to be part of the table
                                              or partition. Defaults to ``false``.
 ============================================ =================================================================
 
 S3 Credentials
 ^^^^^^^^^^^^^^
 
-If you are running Presto on Amazon EC2 using EMR or another facility,
+If you are running Presto on Amazon EC2, using EMR or another facility,
 it is highly recommended that you set ``hive.s3.use-instance-credentials``
 to ``true`` and use IAM Roles for EC2 to govern access to S3. If this is
-the case, your EC2 instances will need to be assigned an IAM Role which
+the case, your EC2 instances need to be assigned an IAM Role which
 grants appropriate access to the data stored in the S3 bucket(s) you wish
 to use. It's also possible to configure an IAM role with ``hive.s3.iam-role``
```

Comment:
Do we want to change "It's" to "It is", too?

Reply:
yeah ... good idea. Changed.